w3f/substrate-benchmarks/roles/gcp/tasks/create.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: Provision GCP credentials
```

Comment:
This looks a bit convoluted to me, why not just, as part of the `requirements` step on CI, store the contents of `GOOGLE_APPLICATION_CREDENTIALS_CONTENT` on a file and then set that file on `gcp_cred_file`?